### DISCUSSION TITLE
The editor is too bad, Almost tortured to death

### DISCUSSION THREAD

**Discussion by wqdygkd** [outline/outline#2847] [Self Hosting]
Sorry, my English is poor

Here are a few questions:

1. The title can not be displayed under the picture. Now, after inserting a picture, there will be a prompt under the picture and it cannot be removed

2. Images cannot be edited after being added

Outline is really good and the UI is beautiful, but I'm not used to this editor and PREFER to write markdown directly

Sorry to bother you.

**Comment by tommoor** [comment]
I have literally no idea what this is a screenshot of...

**Reply by dkkb** [reply]
1. I've just started a new issue on the subject. https://github.com/outline/outline/issues/2854
2. Maybe he wanna resize the image, this is also the feature I'm most interested in.  https://github.com/outline/outline/discussions/1369 This discussion received a lot of votes, will that be our Christmas present?

<img src="https://user-images.githubusercontent.com/82504881/145755940-efb67881-ed5a-4c84-a991-3c3b5818f2c6.png" width=150>

**Comment by tommoor** [comment]
> The title can not be displayed under the picture. Now, after inserting a picture, there will be a prompt under the picture and it cannot be removed

This has been fixed now, it was a consequence of recently moving to an "always editing" mode for new accounts.

> Images cannot be edited after being added

A new "Replace image" option was added to the floating menu

Both of these changes are live on getoutline.com as of yesterday
